core/src/main/java/eu/tsystems/mms/tic/testframework/constants/Browsers.java

```diff
@@ -28,7 +28,5 @@ public class Browsers {
     public static final String ie = "ie";
```

Comment:
Is ie browser still needed?

Reply:
Good question. I think nobody will use it any more but it is still part of Selenium: https://www.selenium.dev/documentation/webdriver/browsers/internet_explorer/